streampark-flink/streampark-flink-connector-plugin/src/main/java/org/apache/streampark/Dummy.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.streampark;
+
+/**
+ * Hello world!
+ *
+ */
+public class Dummy {
+
```

Comment:
Hello, thank you very much for your contribution. What is the `Dummy.java` file used for

Reply:
There are no java files in this project, so put a dummy

Reply:
Can you use package-info.java? Or whatever
![image](https://github.com/user-attachments/assets/034e54da-2b00-46d2-90c4-8079eff549f6)